lib/active_record/connection_adapters/postgis/postgis_database_tasks.rb

```diff
@@ -36,7 +36,7 @@ def create(master_established = false)
 
         # Override to use su_username and su_password
         def establish_master_connection
-          establish_connection(configuration.merge(
+          establish_connection(configuration_hash.merge(
             "database"           => "postgres",
```

Comment:
I would like to change `"database" =>` into `database:`

Reply:
Yes I symbolized these keys on my version